TeamCode/src/main/java/org/firstinspires/ftc/teamcode/commands/groups/ZeroRing.java

```diff
@@ -32,54 +33,57 @@ public ZeroRing(MecanumDriveSubsystem drive, WobbleSubsystem wobbleSystem, Shoot
                 .splineToConstantHeading(new Vector2d(1.0, -60.0), 0.0)
                 .build();
 
-        Vector2d shootPose = traj1.end().vec().plus(new Vector2d(-30.0, 22.0));
+        Vector2d shootPose = traj1.end().vec().plus(new Vector2d(-25.0, 22.0));
 
         Trajectory traj2 = drive.trajectoryBuilder(traj1.end(), true)
                 .lineToConstantHeading(shootPose)
                 .build();
 
-        Vector2d secondWobble = traj2.end().vec().plus(new Vector2d(-12.0, 12.0));
+        //-34 , -24
+//        Vector2d secondWobble = traj2.end().vec().plus(new Vector2d(-14.0, 15.5));
 
         Trajectory traj3 = drive.trajectoryBuilder(traj2.end(), 0.0)
-                .splineToLinearHeading(new Pose2d(secondWobble, 0.0), Math.toRadians(-90.0))
+                .splineToLinearHeading(new Pose2d(-34.0,-24.0, 0.0), Math.toRadians(-90.0))
                 .build();
 
-        Trajectory traj4 = drive.trajectoryBuilder(traj3.end(), false)
-                .splineToConstantHeading(traj3.end().vec().plus(new Vector2d(-2.2, 8.0)), 0.0)
+        Trajectory traj4 = drive.trajectoryBuilder(traj3.end(), 0.0)
+                .splineToSplineHeading(traj3.end().plus(new Pose2d(16.0, 0.0, Math.toRadians(180.0))), 0.0)
+                .splineToConstantHeading(traj1.end().vec().plus(new Vector2d(-12.0, -3.5)), 0.0)
                 .build();
 
-        Trajectory traj5 = drive.trajectoryBuilder(traj4.end(), 0)
-                .splineTo(traj4.end().vec().plus(new Vector2d(16.0, 0.0)), 0.0)
-                .splineToLinearHeading(traj1.end().plus(new Pose2d(-15.0, -12.0, Math.toRadians(180.0))), 0.0)
+        Trajectory traj5 = drive.trajectoryBuilder(traj4.end())
+                .splineToConstantHeading(traj4.end().vec().plus(new Vector2d(-10.0, 16.0)), 0.0)
+//                .splineTo(traj4.end().vec().plus(new Vector2d(-16.0, 16.0)), 0.0)
+                .splineTo(traj4.end().vec().plus(new Vector2d(25.0, 20.0)), 0.0)
```

Comment:
Why not just splineTo a position on the field that is guaranteed to be on the line? Like `(0, -36)`?

Reply:
Would make doing the other autos for parking easier as well. Just spline to that position.